react/src/theme/theme.ts

```diff
@@ -11,6 +11,10 @@ import { layerStyles } from './layerStyles'
 import { textStyles } from './textStyles'
 
 export const theme = extendTheme({
+  config: {
+    initialColorMode: 'light',
+    useSystemColorMode: false,
+  },
```

Comment:
prefer to not pollute the theme with this extra prop, will inject a bunch of data-theme props on the html